test/index.js

```diff
@@ -32,12 +32,22 @@ const changeCwd = function (cwd) {
   return process.chdir.bind(process, originalCwd)
 }
 
+// Move .next from sample project to current directory
+const moveNextDist = function () {
+  // Use copySync because cpx doesn't copy hidden files
+  fs.copySync(`${FIXTURES_DIR}/sample/.next`, `${process.cwd()}/.next`)
+}
+
 // Copy fixture files to the current directory
 const useFixture = async function (fixtureName) {
   const fixtureDir = `${FIXTURES_DIR}/${fixtureName}`
   await pCopy(`${fixtureDir}/**`, process.cwd())
 }
 
+beforeAll(async () => {
+  await npmRun('build', `${FIXTURES_DIR}/sample`)
```

Comment:
Instead of running `npm run build` which runs `next build`, should we just run `next build` directly?

Reply:
with execa? currently this isn't even reliably working

Reply:
Yes. I think removing that additional `npm` step might help make the behavior more reliable.